test/e2e/pages/vl-map-select-action.page.js

```diff
@@ -0,0 +1,59 @@
+const VlMapSelectAction = require('../components/vl-map-select-action');
+const VlMap = require('../components/vl-map');
+const VlMapLayer = require('../components/vl-map-layer');
+const { Page, Config } = require('vl-ui-core').Test;
+const { By } = require('vl-ui-core').Test.Setup;
+
+class VlMapSelectActionPage extends Page {
+
+    async _getSelectAction(selector) {
+        return new VlMapSelectAction(this.driver, selector);
+    }
+
+    async _getLayer(selector) {
+        return new VlMapLayer(this.driver, selector);
+    }
+
+    async _getMap(selector) {
+        return new VlMap(this.driver, selector);
+    }
+
+    async getSelectAction() {
+        return this._getSelectAction('#select-action');
+    }
+
+    async getClusteredSelectAction() {
+        return this._getSelectAction('#select-action-cluster');
+    }
+
+    async clickPointFeature(id) {
+        const layer =  await this._getLayer('#map-layer');
+        const feature = await layer.getFeature(id);
+        const {coordinates} = JSON.parse(feature).geometry;
+        const map = await this._getMap('#map-with-select-action');
+        await map.scrollIntoView();
+        await map.clickOnCoordinates(coordinates);
+        return this._waitForFeatureToBeSelected(id);
+    }
```

Comment:
Waarom `clickPointFeature` en niet `clickFeature`? Ik zie niet meteen point specifieke code?

Reply:
Coordinates van een polygon of line zijn anders, waardoor clickOnCoordinates mogelijk niet werkt